src/common/components/ui/formik/validationSchemas.ts

```diff
@@ -30,7 +30,7 @@ const CLUSTER_NAME_VALID_CHARS_REGEX = /^[a-z0-9-]*$/;
 const SSH_PUBLIC_KEY_REGEX =
   /^(ssh-rsa|ssh-ed25519|ecdsa-[-a-z0-9]*) AAAA[0-9A-Za-z+/]+[=]{0,3}( .+)?$/;
 const DNS_NAME_REGEX = /^([a-z0-9]+(-[a-z0-9]+)*\.)+[a-z]{2,}$/;
-const BASE_DOMAIN_REGEX = /^([a-z0-9]+(-[a-z0-9]+)*)+$/;
```

Comment:
Something doesn't look quite right...
![image](https://user-images.githubusercontent.com/87187179/222701125-658c232f-6874-4d7f-98b1-e97e0931bde4.png)
![image](https://user-images.githubusercontent.com/87187179/222701163-5792a945-97b2-4234-b717-b72edcd03baa.png)

Reply:
I think is correct. @jgyselov why is wrong?

Reply:
No, you are right, I was wrong. :slightly_smiling_face: 
I was thinking about the field in a wrong way.